E2E/pageObjects/game.page.ts

```diff
@@ -5,23 +5,29 @@ const locator = {
     firstUserInList: '//div[contains(@data-testid, "0")]/div[@class="team-list-item-name"]',
     secondUserInList: '//div[contains(@data-testid, "1")]/div[@class="team-list-item-name"]',
     spectatorIcon: '//div[@class="team-list-item-avatar spectator-avatar f-center"]//img[@alt="spectator icon"]',
+    playerDoneIcon: (username: string) => `//div[contains(@data-testid, '${username}')]/img[@alt='player done icon']`,
```

Comment:
If in the future 'div' will change to other element - test(s) will fail. I would suggest instead of '//div' use '//*'